common/src/main/scala/org/neo4j/spark/util/Neo4jOptions.scala

```diff
@@ -1,20 +1,38 @@
 package org.neo4j.spark.util
 
-import org.apache.spark.sql.SaveMode
+import org.apache.spark.sql.{SaveMode, SparkSession}
 import org.neo4j.driver.Config.TrustStrategy
 import org.neo4j.driver._
 
 import java.io.File
 import java.net.URI
+import java.util
 import java.util.UUID
 import java.util.concurrent.TimeUnit
 import scala.collection.JavaConverters._
 
 
-class Neo4jOptions(private val parameters: java.util.Map[String, String]) extends Serializable {
+class Neo4jOptions(private val options: java.util.Map[String, String]) extends Serializable {
   import Neo4jOptions._
   import QueryType._
 
+  private def parameters: util.Map[String, String] = {
```

Comment:
could't find a better way to merge two immutable Maps

Reply:
now it's better!